lib/mini_profiler_rails/railtie.rb

```diff
@@ -75,7 +75,8 @@ def self.initialize!(app)
         next if !should_measure?
 
         current = Rack::MiniProfiler.current
-        description = "Executing action: #{payload[:action]}"
+        controller_name = payload[:controller].sub(/Controller\z/, '').downcase
```

Comment:
I'm not sure if I like the downcasing or not, but that's easy enough to change later.